fe/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1203,4 +1203,19 @@ public boolean convertRandomDistributionToHashDistribution() {
         }
         return hasChanged;
     }
+
+    public void setReplicationNum(Short replicationNum) {
+        if (tableProperty == null) {
+            tableProperty = new TableProperty(new HashMap<String, String>());
+        }
+        tableProperty.getProperties().put(PropertyAnalyzer.PROPERTIES_REPLICATION_NUM, replicationNum.toString());
+    }
+
+    public Short getReplicationNum() {
```

Comment:
I think that may confused user. Because the user doesn't set the replication_num in table level,
however, FeConstants.default_replication_num can be changed by admin, and when user don't modify config for table, he may find the replication_num config in table level changed, return null indicate that we don't set any replication_num config in table level, and then we use FeConstants.default_replication_num